src/rpc/CredentialHelpers.cpp

```diff
@@ -54,4 +58,13 @@ makeSorted(ripple::STArray const& in)
     return out;
 }
 
-}  // namespace rpc
+std::expected<std::set<std::pair<ripple::AccountID, ripple::Slice>>, Status>
+createAuthCredentials(ripple::STArray const& in)
```

Comment:
I think the reason I kept `makeSorted()` is because rippled has the exact same function with the same logic and we wanted to copy it. Although I definitely think it would be better to just rename `createAuthCredentials ` to `makeSorted()` and move all logic inside. WDYT @cindyyan317 ?

Reply:
I think my point is we should not return empty set but want to express there is duplicate in array. If array is empty, `makeSorted` will return empty set but it does not have duplicate. 
I'd rather merge makeSorted logic into `createAuthCredentials` .